common/preprocess.ts

```diff
@@ -62,6 +62,31 @@ const cleanActionStatus = (action, actionStatuses) => {
   return newStatus;
 };
 
+class DonutSector {
+  label: string;
+  color: string;
+  count: number;
+  includeInTotal: boolean;
+  constructor({
+    label,
+    color,
+    includeInTotal,
+  }: {
+    label: string;
+    color: string;
+    includeInTotal: boolean;
+  }) {
+    this.color = color;
+    this.label = label;
+    this.includeInTotal = includeInTotal;
+    this.count = 0;
+  }
+  incr() {
+    this.count++;
+    return this;
+  }
+}
+
```

Comment:
No need to change this but just FYI if you weren't aware, JS classes allow us to define class properties within the constructor so there's no need to define the type on the class and also set the value separately:

```ts
class DonutSector {
  count: number;

  constructor(
    public label: string,
    public color: string,
    public includeInTotal: boolean
  ) {
    this.count = 0;
  }

  incr() {
    this.count++;
    return this;
  }
}

new DonutSector('foo', 'green', false)
```